Utilities/DataLoading/EdFi.SmokeTest.Console/EdFi.SmokeTest.Console.nuspec

```diff
@@ -12,6 +12,6 @@
   </metadata>
   <files>
     <file src="readme.md" target="" />
-    <file src="bin\$configuration$\net48\*.*" target="tools\" />
+    <file src="bin\$configuration$\netcoreapp3.1\*.*" target="tools\"/>
```

Comment:
since you're now calling `dotnet pack` using the .csproj can we just remove the .nuspec files?

Reply:
As Discussed , I am keeping .nuspec files for now